projects/fsastorefrontstyles/scss/base/general.scss

```diff
@@ -1,4 +1,6 @@
-body {
+body,
+html {
+  scroll-behavior: smooth;
```

Comment:
Not related to this task. Enables smooth scrolling on same page when moving from anchor to anchor.